docs/UserGuide.adoc

```diff
@@ -91,6 +91,11 @@ Returns `John Doe` but not `john`.
 * `find Betsy Tim John` +
 Returns Any person having names `Betsy`, `Tim`, or `John`.
 
+== Sort Address Book list : `sort`
```

Comment:
That's good that you have updated the User Guide (UG) to show the newly implemented function that you have created. The UG is the place where the user will go to find out the different features and the usage of it most of the time.